src/components/dialog/dialog.component.tsx

```diff
@@ -305,6 +308,11 @@ export const Dialog = forwardRef<DialogHandle, DialogProps>(
             role={role}
             tabIndex={-1}
             {...contentPadding}
+            backgroundColor={
+              hasGreyBackground
+                ? "var(--colorsUtilityMajor025)"
```

Comment:
**question**: is this definitely the right token? I ask because on the Github issue Luke mentions `colorsUtilityMajor150` as  being correct for the grey background, and you've got 025 rather than 150. No worries if the designs have changed since then (Luke's comment was in August), but I thought it was worth raising this.

Reply:
This was the previous design token being used as the background colour and I also checked in the DS designs and this seems to be the token being used for the grey background.

Reply:
Fair enough. It was [this comment](https://github.com/Sage/carbon/issues/6257#issuecomment-1688299121) that I was referring to, so you may want to check this with Luke - but if you've checked the most recent designs and confirmed you're using the right token as per them then I'm happy enough 👍 (Especially as Harpal will do UX QA anyway.)

Reply:
I've now seen the 025 token in both the Jira ticket and on Figma so a happy to leave this as being correct 👍